packages/airnode-node/config/config.example.json

```diff
@@ -20,7 +20,8 @@
           "value": 3.12,
           "unit": "gwei"
         },
-        "fulfillmentGasLimit": 500000
+        "fulfillmentGasLimit": 500000,
+        "withdrawalRemainder": "0"
```

Comment:
Do we want to have the value as `string` despite the `fulfillmentGasLimit` being number? It feels inconsistent, but we actually want to migrate to strings at some point anyway (see https://api3dao.atlassian.net/browse/AN-711)

Reply:
Same as the other comment: I'm not applying this suggestion as I went with an amount type.